stage_tester/st.py

```diff
@@ -133,12 +169,23 @@ def main():
     if verbose:
         print("Auth settings:", auth)
 
+    if not (args.cluster_list or args.retrieve_results or args.compare_results):
```

Comment:
You can use `any((args.cluster_list, args.retrieve_results, args.compare_results))`

Reply:
or neither? :)

Reply:
fixed :)